docs/sources/panels/visualizations/time-series/graph-time-series-stacking.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "Graph stacked time series"
+keywords = ["grafana", "time series panel", "documentation", "guide", "graph"]
+weight = 400
++++
+
+# Graph stacked time series
+
+> **Note:** This is a beta feature. Time series panel is going to replace the Graph panel in a future release.
+
+This section explains how to use Time series panel field options to control the stacking of the series and illustrates what the stacking options do.
+
+Stacking allows displaying series on top of each other. Be cautious when using stacking in the visualization as it can easily create misleading graphs. You can read more on why stacking may be not the best approach here: [Stacked Area Graphs Are Not Your Friend](https://everydayanalytics.ca/2014/08/stacked-area-graphs-are-not-your-friend.html) 
+
+Use the following field settings to configure your series stacking.
+
+For more information about applying these options, refer to:
+
+- [Configure all fields]({{< relref "../../field-options/configure-all-fields.md" >}})
+- [Configure specific fields]({{< relref "../../field-options/configure-specific-fields.md" >}})
+
+Some field options will not affect the visualization until you click outside of the field option box you are editing or press Enter.
+
+## Stack series
+
+Turn series stacking on/off.
+
+![Stack series editor](/img/docs/time-series-panel/stack-series-editor-8-0.png)
+
+### Off
+
+Disable series stacking.
+
+![No series stacking example](/img/docs/time-series-panel/stacking-off-8-0.png)
+
+### Normal
+
+Enable stacking series on top of each other.
+
+![Normal series stacking example](/img/docs/time-series-panel/stacking-normal-8-0.png)
+
+## Stack series in groups
+
+Stacking group option is only available as an override.
```

Comment:
```suggestion
The stacking group option is only available as an override.
```

Reply:
Why are we calling it "stacking group" if those words do not appear in the UI?

Reply:
They do, as a placeholder in the stacking group input